client/reader/stream/post-unavailable.jsx

```diff
@@ -36,7 +30,10 @@ class PostUnavailable extends PureComponent {
 				</div>
 
 				<div className="reader__post-excerpt">
-					<p>{ errorMessage }</p>
+					<p>
```

Comment:
Hmm in the "houseofthedragon" example search, I'm not seeing this is-error card displayed...

Reply:
oh nvm, this would be hidden for 404 type errors 👍

Reply:
I hacked it to render an error page even for 404 errors by commenting out[ line 14](https://github.com/Automattic/wp-calypso/pull/68991/files#diff-66f9ea774703b7a8a69a8d8b07d3fc5ab74998b47969e7062968692a2cd5524eR14). It looks like this, I think a bit strange, but fair enough, it is an error state. (screenshot is zoomed out)
<img width="550" alt="Screen Shot 2022-10-17 at 11 52 39 am" src="https://user-images.githubusercontent.com/22446385/196072669-10a9d68e-fa8a-4bb1-bb5e-e1cfd4b499b4.png">